dfmoco2ur-ui/src/components/LoggEntry.js

```diff
@@ -0,0 +1,28 @@
+
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import Container from "@material-ui/core/Container";
+import ControlCanvas from "./ControlCanvas";
+import Button from '@material-ui/core/Button';
+import LoggView from './LoggView';
+
+const useStyles = makeStyles((theme) => ({
+    paper: {
+      padding: "6px 16px",
+    },
+    secondaryTail: {
+      backgroundColor: theme.palette.secondary.main,
+    },
+  }));
+
+  export default function ControlGrid() {
```

Comment:
Komponente entsprechend dem Dateinamen benennen.

```suggestion
  export default function LogEntry() {
```